queries/formInstances.ts

```diff
@@ -92,3 +92,16 @@ export const getFormInstances = async (formLabel: string, next) => {
   const result = { instances: instance_values };
   return result;
 };
+
+export const deleteFormInstance = async function (instanceUri: string, next) {
```

Comment:
i think we need to be more careful here. this will delete every triple starting from the instance, but the form may actually contain way more than that (triples in the other direction, nested instances, ...) 

I would use the construct query to build all triples of the form and delete that way, wdyt?

Reply:
Ah yes, didn't think about that, but wouldn't it also work if I just delete all triples from both directions? All triples should be connected to the uri, no?